synapse/tests/test_lib_stormtypes.py

```diff
@@ -117,3 +117,19 @@ class StormTypesTest(s_test.SynTest):
 
             self.eq(gotn[0][1]['type'], 'foo:bar')
             self.eq(gotn[0][1]['data']['baz'], 'faz')
+
+    async def test_storm_node_repr(self):
+
+        text = '''
+            [ inet:ipv4=1.2.3.4 :loc=us]
```

Comment:
we should be using test types here

Reply:
I'm kinda 50/50 on that.  With the number of times we tell people that the unit test code is full of decent examples, having this be something they understand ( that an ipv4 is an integer but is displayed differently ) makes it more compelling than some abstract type.  ( also, very low risk of that node and it's props changing :D )